student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(id, name, age, parent_permission, classroom)
+    super(id, name, age, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+end
```

Comment:
- [ ] In the initialize constructor method, the parameters - `name` and `parent_permission` should have default values like so 👇🏾 

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Kindly fix this